### PR TITLE
ADDED swift test file for syntax highlighting

### PR DESCRIPTION
**Pull request by AkshatGadhwal**
Added a swift file containing around 200 lines of code. Tried to include as much syntax as I could. 
issue #1213

### CODE REVIEW

File: tests/syntax-tests/source/Swift/test.swift [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+    {
+        returnValue = true
+    }
+    return returnValue
+}
+
+  let friend1 = Friend(name: "Sergey", age: 35)
+        let friend2 = Friend(name: "Sergey", age: 30)
+        
+        print("\Compare Friend object. Same person? (friend1 == friend2)")
```
Comment (sharkdp) [inline tests/syntax-tests/source/Swift/test.swift:40]: what is `\C`?

Comment (AkshatGadhwal) [reply tests/syntax-tests/source/Swift/test.swift]: Oo sorry, there is a typo.
I will correct it.

**Comment by sharkdp** [comment]
The lines have wrong line endings (that's why the tests fail). Have you cloned the `bat` repository again after I merged #1254?

**Comment by AkshatGadhwal** [comment]
> The lines have wrong line endings (that's why the tests fail). Have you cloned the `bat` repository again after I merged #1254?

Yes i cloned it again.

**Pull request merged**